mscore/prefsdialog.ui

```diff
@@ -2498,6 +2498,9 @@
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
+            <property name="currentText">
+             <string notr="true" extracomment="Dummy values. Used for clarity in the Designer. Translated elsewhere.">Percentage</string>
+            </property>
```

Comment:
That's just one of the 4 strings

Reply:
and you added it rather than modifying it?

Reply:
I don't understand what you are saying. I reset my changes to this file and then went to QtDesigner, disabled translatable in the QComboBox and added a comment.

Reply:
Apparently not.

Reply:
Done :pray: